docs/zh/latest/plugins/http-logger.md

```diff
@@ -27,7 +27,7 @@ title: http-logger
 
 这将提供将 Log 数据请求作为JSON对象发送到监视工具和其他 HTTP 服务器的功能。
```

Comment:
```suggestion
这将提供将 Log 数据请求作为 JSON 对象发送到监视工具和其他 HTTP 服务器的功能。
```
Need black space between Chinese and English.

Refer to https://github.com/sparanoid/chinese-copywriting-guidelines

Reply:
done